src/Symfony/Component/Secret/AbstractVault.php

```diff
@@ -9,7 +9,9 @@
  * file that was distributed with this source code.
  */
 
-namespace Symfony\Bundle\FrameworkBundle\Secrets;
```

Comment:
Instead of moving you should copy & extend those new files by old ones to have BC.

Reply:
It is not a problem that the component is dependent on the FramworkBundle?

Reply:
It is, but we can do differently.
The deprecated framework-bundle classes should extend the new component’s classes, or be aliased to them (see e.g. https://github.com/symfony/symfony/blob/fdfa40e33885b6d42cf8c05b60536e2f10cb3118/src/Symfony/Component/Messenger/Transport/AmqpExt/AmqpTransport.php#L18).

Reply:
All classes are flagged `@internal`. As long as `symfony/secret` is a required dependency we don't need to use this trick.

But if we decide to make `synfony/secret` optional (see my comment https://github.com/symfony/symfony/pull/45571#discussion_r815470542), in that case, we should keep a BC layer in the FrameworkBundle